layers/generate_layers.py

```diff
@@ -345,6 +355,120 @@ def makeLocalAuthorityDistricts():
         f.write(json.dumps(gj))
 
 
+# You have to manually download the GeoJSON file from https://geoportal.statistics.gov.uk/datasets/ons::output-areas-2021-boundaries-ew-bgc/explore and pass in the path here (until we can automate this)
+def makeCensusOutputAreas(raw_boundaries_path):
+    tmp = "tmp_census_output_areas"
+    os.makedirs(tmp, exist_ok=True)
+
+    # Build up a dictionary from OA code to properties we want
```

Comment:
I chose not to use geopandas / regular pandas for the operations here, since plain Python is just as easy to work with. The others require external Python dependencies, and unfortunately there's not one simple answer to managing environments. I started trying https://python-poetry.org/ here, but hit problems with numpy versions and python 3.8. I'll try again later, but for now, going to keep avoiding any external python dependencies

Reply:
Ok, makes sense